### DISCUSSION TITLE
Where to store input data

### DISCUSSION THREAD

**Discussion by wolfidan** [openradar/erad2022#4] [Q&A]
Just a question regarding where to store the input data used in the notebook exercices.
Since Binder relies on the github, shall we store the files on github as well provided they are not too big?
Sorry if the question is stupid, I have no experience with notebooks in the cloud yet.

**Comment by mgrover1** [comment]
@wolfidan this is a great question - it would make sense to keep data on the repo where possible, and only download when we need bigger datasets. Do you have any larger datasets in mind?

**Reply by wolfidan** [reply]
For pyart I don't think we would work with very large datasets (with files larger than github's 100 mB limit) but I don't know how it is for other modules (baltrad, lrose). I also agree that it would be good to store the data in a input_data (for example) directory on the repo and then separate the data by module.

**Reply by mgrover1** [reply]
This spring, we added the `get_test_data()` function to retrieve data from an FTP server 

See one of the examples using it [here](https://arm-doe.github.io/pyart/examples/plotting/plot_three_panel_gridmapdisplay.html#sphx-glr-examples-plotting-plot-three-panel-gridmapdisplay-py) so we should be good on the Py-ART side.

We can add a `/data` directory at the top level of the directory that the different modules can share!
